### PR TITLE
Get all

### PR DESCRIPTION
**Pull request by sahedw**

### CODE REVIEW

File: frontend/src/components/FoodSpotCard.tsx [hunk 1]
```diff
@@ -0,0 +1,19 @@
...
+    return (
+        <div className={"foodspot-card-container"}>
+            <h3>{props.foodSpot.name}</h3>
+            <img className={`card-image`} src={`${props.foodSpot.category}.png`} alt="food image"/>
+            <p>{props.foodSpot.address}</p>
+        </div>
+    );
+}
+
+export default FoodSpotCard;
```
Comment (Pia0912) [inline frontend/src/components/FoodSpotCard.tsx:19]: you can also move this to line 7, and delete line 19:
```
export default function FoodSpotCard 
```

File: backend/pom.xml [hunk 2]
```diff
@@ -39,11 +38,36 @@
 			<artifactId>spring-boot-starter-test</artifactId>
 			<scope>test</scope>
 		</dependency>
+		<dependency>
+			<groupId>de.flapdoodle.embed</groupId>
+			<artifactId>de.flapdoodle.embed.mongo.spring30x</artifactId>
```
Comment (Pia0912) [inline backend/pom.xml:43]: funny formatting (at least in the github view) 
If you did not know already, with "Option+Command+L" you can automatically format your code

**Review by Pia0912 — APPROVED** [review]
LGTM! 🚀🍱

File: backend/pom.xml (hunk 2 above)
Comment (sahedw) [reply backend/pom.xml:43]: Appreciate it, just looks like that in GitHub. Weird.

**Pull request merged**